Lib/test/test_uuid.py

```diff
@@ -1,3 +1,4 @@
+import threading
```

Comment:
```suggestion
import threading  # XXX: RUSTPYTHON; to check `_at_fork_reinit`
```
not to remove it when update to next CPython version